default/stringtables/en.txt

```diff
@@ -4442,6 +4442,7 @@ Locked
 ALL_POLICY_AVAILABILITY_FILTERS_BLOCKING_PROMPT
 Toggle the category and availability filters to show policies.
 
+
```

Comment:
If you have a fix like this, it's a lot better to put it into a separate pull request that fixes just that, so I can just accept it. If it's added to an unrelated pull request like this, I have to separately pull and commit it or wait until the rest of the pull request is OK...